src/components/Tabs/Tabs.mdx

```diff
@@ -128,6 +132,15 @@ the <Link href="#children-components" target="_self">Children Components</Link>
 To view and test this in Storybook, go to the "Canvas" tab and change the
 viewport in the Storybook toolbar at the top of the page.
 
+## Scrollable Desktop Tab Set
+
+For desktop viewports (600px and greater), the full tab set scrolls left/right when
+the width of the tab set is greater than the width of the viewport.
+
+To scroll horizontally, use the `shift + scroll wheel` combination.
```

Comment:
Nice! Did not know this. I usually scroll horizontally on the mac trackpad but this is so useful.